### ISSUE TITLE
Aligning on variable assignment

### ISSUE THREAD

**Issue by Aedaeum** [chemoish/atom-valign#5]
This is a great plugin. I have only one suggestion

from
```
var myvar1 = sometext,
  myvar2 = sometextlonger,
  myvarlonger3 = somettext,
  myvar4 = moretext;
```

to
```
var myvar1       = sometext,
    myvar2       = sometextlonger,
    myvarlonger3 = somettext,
    myvar4       = moretext;
```

**Comment by chemoish** [comment]
Hi @Aedaeum,

Thanks for the feature request. This annoys me like no other, but I have been manually aligning it since I moved from coffee => javascript.

I will get on this as soon as I figure out whether or not I want to refactor this package to be more configurable.

**Comment by Aedaeum** [comment]
Thanks for the quick response.
I will definitely be looking forward to this.

**Comment by chemoish** [comment]
Now that I am thinking about it, it is getting pretty complex.

I might start with something then we can tweak it. I wonder if my friend has tackled this problem yet.

```
var func = function () {};
# comment here
var one = 1,
      two = 2;
var three = 3,
      four = 4;

**Comment by Aedaeum** [comment]
I agree that sticking to each var block separated by a new line is preferable.

**Comment by chemoish** [comment]
@Aedaeum, lots of breaking changes with all the code modifications. Though the basic feature support for var alignment is in there. It is a bit finicky because the implementation was even more complicated than I stated before.

Please let me know if you run into any other bugs caused by the refactor.

Fixed by 6ed73e85ac27f46ffe4d539102f5d6c4ee44ae4a
